keps/sig-storage/2639-secret-protection/kep.yaml

```diff
@@ -0,0 +1,51 @@
+title: secret protection
+kep-number: 2639
+authors:
+  - "@mkimuram"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-storage
+status: provisional
+creation-date: 2021-04-20
+reviewers:
+  - TBD
+approvers:
+  - TBD
+#prr-approvers:
+# - TBD
+#see-also:
+#  - "/keps/sig-aaa/1234-we-heard-you-like-keps"
+#  - "/keps/sig-bbb/2345-everyone-gets-a-kep"
+#replaces:
+#  - "/keps/sig-ccc/3456-replaced-kep"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.22"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.22"
+  beta: "v1.23"
+  stable: "v1.24"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: SecretInUseProtection
+    components:
+      - kube-controller-manager (in-tree)
+      - secret-protection-controllerr (in-tree)
+      - storageobjectinuseprotection admission plugin (in-tree)
+      - secret-protection-volumesnapshot-controller (out-of-tree)
+      - storageobjectinuseprotection volumesnapshot admission plugin (out-of-tree)
```

Comment:
Similar comments as earlier.

Reply:
Tried to nest the component, but it caused a marshal error in kep.yaml. So, added a note with (). Should we rather just keep kube-controller-manager and delete others?